### DISCUSSION TITLE
Npm Install

### DISCUSSION THREAD

**Discussion by Oyebolade** [rupali-codes/LinksHub#184] [Q&A]
This is what my npm install display, can anyone help me out?

PS C:\Users\OYEBOLADE OLADOKUN\Desktop\OpenSource\LinksHub> npm install
npm ERR! code ERR_SOCKET_TIMEOUT
npm ERR! network Socket timeout
npm ERR! network This is a problem related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\OYEBOLADE OLADOKUN\AppData\Local\npm-cache\_logs\2023-02-11T01_14_55_841Z-debug-0.log
PS C:\Users\OYEBOLADE OLADOKUN\Desktop\OpenSource\LinksHub>

**Comment by rupali-codes** [comment]
It's you network issue. You should try now

**Reply by Oyebolade** [reply]
All right, I will surely do that.
Thanks

**Reply by Oyebolade** [reply]
All right, I will surely do that.
Thanks

**Reply by Oyebolade** [reply]
All right, I will surely do that.
Thanks

**Comment by Oyebolade** [comment]
All right, I will surely do that.
Thanks

**Comment by itsaadish** [comment]
Hi I have done npm install and when i run npm dev I am getting this error please help
> LinksHub@0.1.0 dev /Users/aadishsharma/Documents/Personal/LinksHub
> next dev

/Users/aadishsharma/Documents/Personal/LinksHub/node_modules/next/dist/cli/next-dev.js:309
            showAll: args["--show-all"] ?? false,
                                         ^

SyntaxError: Unexpected token '?'

**Reply by itsaadish** [reply]
It is resolved now
